src/data/fintype/fin.lean

```diff
@@ -45,4 +47,20 @@ begin
   { rintro ⟨i, hi, rfl⟩, simpa },
 end
 
+lemma card_filter_univ_fin (p : (fin (n + 1)) → Prop) [decidable_pred p] :
```

Comment:
```suggestion
lemma card_filter_univ_succ (p : fin (n + 1) → Prop) [decidable_pred p] :
```
since you're already in the fin namespace, and this is consistent with the name `fin.univ_succ`.